src/components/ParaNav.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import $box from "@/styles/Box.module.css";
+import $button from "@/styles/Button.module.css";
+import Link from "next/link";
+import { useRouter } from "next/router";
+
+const ParaNav = () => {
+  const { asPath } = useRouter();
+  const modifyPaths = (path: string, toPath: string) => {
+    // If we are already at that path, we want to return to the benchmark
+    // If not, we move to the new path
+    if (path.includes(toPath)) {
+      return path.slice(0, path.lastIndexOf("/"));
+    }
+    if (path.includes("/studentprofile") || path.includes("/instructions")) {
+      return path.slice(0, path.lastIndexOf("/")) + toPath;
+    }
+    return path + toPath;
+  };
```

Comment:
sorry, not really understanding what's going on here

Reply:
Basically, the two para nav buttons should either route to `/benchmarks/[id]/route` or return back to `/benchmarks/[id]` if we're already on that page. But looking at it again, there's a much simpler solution using router.query.